pmlc/util/ids.h

```diff
@@ -0,0 +1,13 @@
+// Copyright 2020 Intel Corporation
+#pragma once
+
+// A collection of well-known identifiers used in codegen.
+
+namespace pmlc {
```

Comment:
`pmlc::util`

Reply:
Done.  (Seems unnecessary to me, tho -- lots of `util::` sprinkled everywhere for no obvious value.)

Reply:
It's just consistency with the directory structure. I'm fine with flattening the namespace so it's not so deep, but I think we should do that with buy-in from others and that should be its own NFC PR.